modules/rosa-create-cluster-admins.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// getting_started_rosa/creating-first-rosa-cluster.adoc
+
+
+[id="rosa-create-dedicated-cluster-admins.adoc"]
+= Granting cluster-admin access
+
+As the user that created the cluster, add the `cluster-admin` user role to your account to have the maximum administrator privileges. These privileges are not automatically assigned to your user account when you create the cluster.
+
+Additionally, only the user who created the cluster can grant cluster access to other `cluster-admin` or `dedicated-admin` users. `dedicated-admin` users have fewer privileges. As a best practice, limit the number of `cluster-admin` users.
```

Comment:
Per ISG, I would reword the second sentence so that it does not start lowercase. Perhaps:
> Users with `dedicated-admin` access have fewer privileges.

Reply:
Limit the number to what?

Reply:
Done. Limit to as few as possible. One of the things we're trying to do is get better guidance around cluster-admin versus dedicated admins on OSD, and the specific differences between the privileges. Future enhance,ent. This is the best I can do for now.